ref(azure): Change usage of bold, add final step for successful login (#3302)
getsentry/sentry-docs: src/docs/product/accounts/sso/azure-sso.mdx

    - First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`

    - Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
    

>     - For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).

Reviewer: This doesn't make structural sense as a bullet of step since it's not one of AzureAD keys. I've tried to format it so that it remains indented--so that it's clear the information relates to the larger step--but just without a bullet point. 

```suggestion
      For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
```

   ![Map Identity with provider attributes](azure-map-identity-provider-attributes.png)
   

> 1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page.

Reviewer: There's a missing connection between this lead in sentence and the bullet points that follow. I've made a suggestion below (and edited to the bullet points to make it work), but you might know a better way to link these. 

```suggestion
1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
```

Author: Works for me 👍

>    - The `Login URL` will be used for SP-initiated SSO. You can share it with the users in your organization.
>    
>    - Scroll down to the bottom. Ensure that `Require SSO` is checked if you want to enforce logging in with Okta.
>    
>    - Set a `Default Role` for new SSO users. Selecting `Member` should cover most use-cases.
>    
>    - If you made changes, click **Save Settings** and that's all for the setup!

Reviewer: Without seeing this page, I've made the assumption that these are field labels that you're talking about so they'd take the UI element styling. 

```suggestion
   - You can share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
   
   - Scroll down to the bottom and ensure that "Require SSO" is checked if you want to enforce logging in with Okta.
   
   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use-cases.
   
   - If you made changes, click "Save Settings" to complete your setup.
```
